src/assets/crisistextline/sass/partials/01-base/_base.scss

```diff
@@ -9,6 +9,19 @@
     background-image: image-url('loader.gif') no-repeat 95%;
 }
 
+body {
+    background-color: $color-light;
+}
+
+#main-wrapper {
+    background-color: white;
+}
+
+div[role="main"] {
+    @include calc(width, 100% - #{$width-sidebar-left});
+    float: right;
+}
+
```

Comment:
it seems like these changes would make more sense in `/03-components/_main.scss` (my comment in line 85 there notwithstanding)? it's been my ultimate aim to refactor how we break the sass down, but for now, `_base.scss` is more for generic classes, and `_main.scss` is for this kind of color and positioning stuff